modules/index-service/src/main/java/org/opencastproject/index/service/impl/IndexServiceImpl.java

```diff
@@ -1347,6 +1374,7 @@ public MetadataList updateEventMetadata(String id, MetadataList metadataList, El
         break;
       case ARCHIVE:
         assetManager.takeSnapshot(mediaPackage);
+        updatePublication(mediaPackage);
```

Comment:
Is it correct to take the snapshot **before** updating the publications? 🤔

Reply:
No idea. It seemed reasonable to first make sure data is saved before distributing it. But the default workflows do it the other way around it seems.